scene/animation/animation_tree.cpp

```diff
@@ -1159,7 +1159,7 @@ void AnimationTree::_process_graph(real_t p_delta) {
 									}
 								}
 							} else {
-								if (prev_time > a->get_length()) {
+								if (prev_time > (double)a->get_length()) {
```

Comment:
Why? I don't think that an explicit cast in this comparison does anything. Or is it to avoid warnings?